### DISCUSSION TITLE
AP mode on ESP 32 not working and can't find docker Hub like in video.

### DISCUSSION THREAD

**Discussion by HungDK763** [makerspet/support#14] [Support Q&A]
**1: AP mode on ESP 32 not working** 
I loaded the program to the esp 32, I was able to find the wifi and connect to it.
![ừ](https://github.com/user-attachments/assets/79462a1a-4864-4f61-bbea-f4540e139959)
However, when I access IP 192.168.4.1, I cannot access it and each time I access it, the wifi disappears. 
![No_connect](https://github.com/user-attachments/assets/23aa26f1-b780-478e-8a89-5b7234fceafd)
**2: can't find docker Hub like in video.**
when i tried to follow the video, the console said that it couldn't find the same item in Docker Hub.

![docker not find](https://github.com/user-attachments/assets/87bee794-584e-4d1d-97c0-e9b47b6a9016)
![Docker HUb ](https://github.com/user-attachments/assets/f5145174-56ad-49e7-aee5-5d623f6905b5)

**Comment by makerspet** [comment]
Hi, the video is too old. All up-to-date instructions are here https://github.com/kaiaai/kaiaai
For example, the image pull command is 
```
docker pull kaiaai/kaiaai:humble
```
Regarding web configuration
- can you post the output of your Arduino serial debug window
- which ESP32 model are you using?
- make sure to copy libraries from https://github.com/kaiaai/firmware/tree/iron/libraries into your Arduino folder
FYI, the AP mode and web configuration work very well for me. I use it almost every day.
